seqr/views/utils/test_utils.py

```diff
@@ -358,9 +358,6 @@ def get_workspaces_side_effect(user):
 
 
 class AnvilAuthenticationTestCase(AuthenticationTestCase):
-    @classmethod
-    def setUpTestData(cls):
-        _initialize_users(cls)
```

Comment:
Since this `setUpTestData` is deleted, the `_initialize_users` is used only once and can be deleted.